src/common/constants.js

```diff
@@ -80,10 +80,17 @@ export const default_server_url = 'green.binaryws.com'
 
 export const live_chat_key = 'live_chat_redirection'
 
+export const fb_non_eu_url = 'https://www.facebook.com/derivdotcom'
```

Comment:
This looks like a duplication of that on line 86. Can we just use that instead? Same for the Instagram and Twitter ones.